generators/csharp/sdk/versions.yml

```diff
@@ -6,6 +6,22 @@
 #         The C# SDK now uses forward-compatible enums which are not compatible with the previously generated enums.
 #         Set `enable-forward-compatible-enums` to `false` in the configuration to generate the old enums.
 #   irVersion: 53
+- version: 1.13.0-rc0
```

Comment:
Should we keep this in `1.12.0-rcX` until we're out of the RC state? Otherwise we'll have release candidates across multiple minor versions.